src/frontend/src/pages/FinancePage/ReimbursementRequestForm/ReimbursementFormView.tsx

```diff
@@ -224,7 +224,7 @@ const ReimbursementRequestFormView: React.FC<ReimbursementRequestFormViewProps>
                       });
                     } else {
                       toast.error('File must be less than 1 MB', 5000);
-                      document.getElementById('receipt-image')!.value = '';
+                      document.getElementById('receipt-image')!.innerHTML = '';
```

Comment:
this technically doesn't work cause it doesn't clear the image that's being stored in the input this way, but I'ma go ahead and merge, we can fix this later